pyquil/quil.py

```diff
@@ -82,6 +82,10 @@
     _convert_to_calibration_match,
 )
 
+if np.lib.NumpyVersion(np.__version__) >= "2.0.0b1":
+    # prevents floats from being printed as `np.float(1.234)`
+    np.set_printoptions(legacy="1.21")
+
```

Comment:
Is there a better way to do this, rather than having this in all file which might call `to_quil`?